common.sh

```diff
@@ -98,6 +104,10 @@ export REGISTRY_PASS=${REGISTRY_PASS:-ocp-pass}
 export REGISTRY_DIR=${REGISTRY_DIR:-$WORKING_DIR/registry}
 export REGISTRY_CREDS=${REGISTRY_CREDS:-$HOME/private-mirror-${CLUSTER_NAME}.json}
 export REGISTRY_CRT=registry.2.crt
+export REGISTRY_BACKEND=${REGISTRY_BACKEND:-"podman"}
+if [[ ${REGISTRY_BACKEND} = "quay" ]]; then
+    export LOCAL_REGISTRY_PORT=8443
```

Comment:
Why not using the usual 5000 port? The backend may change, buy the way the service is exposed should remain the same

Reply:
We could use the same port although having different ports has been useful in testing to easily know which backend is being used. I'd prefer to keep them different so its obvious.